### PR TITLE
doc: Optimizing AWS Lambda Documentation

### PR DESCRIPTION
**Pull request by nnennandukwe**
## About the changes

Closes #

## Discussion points

Updates according to our revision doc for our horizontal content pieces.

### CODE REVIEW

File: website/docs/feature-flag-tutorials/serverless/implementing-feature-flags-in-aws-lambda.md [hunk 1]
```diff
@@ -1,142 +1,174 @@
 ---
-title: How to implement feature flags in a serverless environment using AWS Lambda
+title: How to Implement Feature Flags in a Serverless Environment using AWS Lambda
+description: New to feature flags in serverless environments? This guide will walk you through a practical example using the Unleash Node.js SDK in AWS Lambda.
 slug: /feature-flag-tutorials/serverless/lambda
 ---
 
-For developers new to feature flags in serverless environments, this guide will walk you through practical examples using the [Unleash Node.js SDK](https://github.com/Unleash/unleash-client-node) in [AWS Lambda](https://aws.amazon.com/lambda/). These concepts can be easily adapted to other serverless solutions like [Google Cloud Functions](https://cloud.google.com/functions). Having feature flagging capabilities available inside of your serverless functions allows you to validate new parts of your serverless functions and stay in control of the feature exposure. 
+With feature flagging capabilities in your serverless functions, you can validate new parts of your serverless functions. This gives you complete control over feature exposure and simplifies serverless function versioning management.
```
Comment (ferrantim) [inline website/docs/feature-flag-tutorials/serverless/implementing-feature-flags-in-aws-lambda.md:7]: the first sentences uses "serverless functions" twice which feels repetitive. Can you update?

File: website/docs/feature-flag-tutorials/serverless/implementing-feature-flags-in-aws-lambda.md [hunk 2]
```diff
@@ -1,142 +1,174 @@
...
 
+For developers new to feature flags in serverless environments, this guide will walk you through practical examples using the [Unleash Node.js SDK](https://github.com/Unleash/unleash-client-node) in [AWS Lambda](https://aws.amazon.com/lambda/). These concepts can be easily adapted to other serverless solutions like [Google Cloud Functions](https://cloud.google.com/functions).
+
+## Scenarios for AWS Lambda feature flags
+
+-   If you make a breaking change to a serverless function (e.g., new params), you usually need to create a new version with an appropriate URL routing scheme
+-   But sometimes you may need to add a non-breaking change, signature-wise, that may still have unwanted side effects for some consumers
+    -   Give a concrete example
```
Comment (ferrantim) [inline website/docs/feature-flag-tutorials/serverless/implementing-feature-flags-in-aws-lambda.md:15]: Is this section finished?  Looks like placeholder content still

File: website/docs/feature-flag-tutorials/serverless/implementing-feature-flags-in-aws-lambda.md [hunk 3]
```diff
@@ -1,142 +1,174 @@
...
+    destroyWithFlush,
+} from "unleash-client";
 let unleash;
 
 async function init() {
     if (!unleash) {
         unleash = await startUnleash({
-            url: 'https://sandbox.getunleash.io/enterprise/api/',
-            appName: 'lambda-example-app',
+            url: "https://sandbox.getunleash.io/enterprise/api/",
```
Comment (alvinometric) [inline website/docs/feature-flag-tutorials/serverless/implementing-feature-flags-in-aws-lambda.md:48]: Probably put `<YOUR_UNLEASH_URL>` here like we do elsewhere?

File: website/docs/feature-flag-tutorials/serverless/implementing-feature-flags-in-aws-lambda.md [hunk 4]
```diff
@@ -1,142 +1,174 @@
...
 
-process.on('SIGTERM', async () => {
-    console.info('[runtime] SIGTERM received');
+process.on("SIGTERM", async () => {
+    console.info("[runtime] SIGTERM received");
 
-    if(unleash) {
+    if (unleash) {
         await destroyWithFlush();
         unleash = undefined;
```
Comment (alvinometric) [inline website/docs/feature-flag-tutorials/serverless/implementing-feature-flags-in-aws-lambda.md:131]: I would rather set this to `null`

File: website/docs/feature-flag-tutorials/serverless/implementing-feature-flags-in-aws-lambda.md (hunk 2 above)
Comment (nnennandukwe) [reply website/docs/feature-flag-tutorials/serverless/implementing-feature-flags-in-aws-lambda.md:15]: @ivarconr do we have an example for this? I'm assuming "signature-wise" is referring to code signing, but not sure.

**Pull request merged**